packages/libs/error-stack/src/hook/mod.rs

```diff
@@ -16,6 +16,8 @@ type RwLock<T> = std::sync::RwLock<T>;
 type RwLock<T> = spin::rwlock::RwLock<T>;
 
 static FMT_HOOK: RwLock<Hooks> = RwLock::new(Hooks { inner: Vec::new() });
```

Comment:
We probably also fully qualify the path to `Hooks` here as well or import them as `FmtHooks`. Actually, as they are internal only, I'd prefer renaming both `Hooks` structs to `FmtHooks` and `SerdeHooks`

Reply:
fixed in [83edbf2](https://github.com/hashintel/hash/pull/1558/commits/83edbf2a5191c8c8b38615601044655aea37277f)